distribution/client/src/lib/update.ts

```diff
@@ -22,6 +22,8 @@ export interface FileOptions {
   flag?: string,
 };
 
+interface Foo {}
```

Comment:
I assume that this is here just to fail the build on purpose?

Reply:
yes, but does not work, hence the WIP state :cry: